extensions/ql-vscode/test/vscode-tests/no-workspace/telemetry.test.ts

```diff
@@ -574,6 +573,16 @@ describe("telemetry reporting", () => {
     await wait(100);
   }
 
+  async function setTelemetryLevel(section: string, value: string | undefined) {
+    await workspace
+      .getConfiguration(section)
+      .update("telemetryLevel", value, ConfigurationTarget.Global);
+
+    // Need to wait some time since the onDidChangeConfiguration listeners fire
+    // asynchronously. Must ensure they to complete in order to have a successful test.
+    await wait(100);
```

Comment:
An observation, but not introduced in this PR so doesn't need to be address now: We seem to call `enableTelemetry` and `setTelemetry` sequentially. They're both updating the config and then wait for 100ms. I wonder if we could merge the calls into one and only wait once. It would shave off some waiting timing.

Reply:
Good point! 👀 I'll leave things as-is for now, since there'll likely be more changes to the tests anyway, if we manage to upgrade the telemetry package 🤞🏽